skill.json

```diff
@@ -1,38 +1,36 @@
 {
```

Comment:
Another duplicate of `locale/en-us/skill.json`? Though this might be worth keeping for backwards-compat with any tools looking in the repo root

Reply:
This one was intentional for that very reason